src/app/layout.tsx

```diff
@@ -14,11 +15,15 @@ import { metropolis } from "./fonts/Metropolis/metropolis";
 
 const inter = Inter({ subsets: ["latin"], variable: "--font-inter" });
 
-export async function generateMetadata() {
+export async function generateMetadata(): Promise<Metadata> {
   const l10n = getL10n();
   return {
     title: l10n.getString("brand-fx-monitor"),
     description: l10n.getString("meta-desc-2"),
+    metadataBase:
+      typeof process.env.SERVER_URL === "string"
+        ? new URL(process.env.SERVER_URL)
```

Comment:
Not sure when/if this could happen, but if your `SERVER_URL` is an _empty_ string, this will throw a TypeError on the `URL()` constructor call:

```js
let serverUrl = "";
new URL(serverUrl);

/*
Uncaught TypeError: Failed to construct 'URL': Invalid URL
    at <anonymous>:1:1
*/
```

Reply:
`SERVER_URL` is defined in `.env-dist`, if it's not set I think we probably do want to throw here since lots of things won't work correctly. It's not something we can reasonable handle and keep the server functioning, I don't think.

Reply:
If you want to explicitly catch I think that's fine, but I would re-throw. Up to you I think @Vinnl - I prefer to not handle exceptions unless we are sure we really can handle them (e.g. "swallowing" the exception by just logging it and moving on would just lead to later failures, so we're not really "handling" it)